### PR TITLE
fix(stylus): make header color less deep to match text

### PR DESCRIPTION
**Pull request by GitMuslim**
## 🔧 What does this fix? 🔧

# Original Stylus
image

# Before
image

# After
image

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [x] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

**Comment by GitMuslim** [comment]
@isabelroses

File: styles/stylus/catppuccin.user.css [hunk 1]
```diff
@@ -199,4 +199,3 @@
     #catppuccin(@darkFlavor, @accentColor);
   }
 }
-// vim:ft=less
```
Comment (isabelroses) [inline styles/stylus/catppuccin.user.css:202]: can you revert this change

Comment (GitMuslim) [reply styles/stylus/catppuccin.user.css]: Done

**Review by isabelroses — APPROVED** [review]
